VagrantProvisionVars.sh

```diff
@@ -27,10 +24,5 @@ export PROJ_VERSION=7.2.1
 export POSTGRESQL_VERSION=13
 export POSTGRESQL_VERSION_DOTLESS="$(echo "$POSTGRESQL_VERSION" | awk '{ gsub(/\./, ""); print substr($0, 1, 2) }')"
 
-# FGDB 1.5 is required to compile using g++ >= 5.1
-# https://trac.osgeo.org/gdal/wiki/FileGDB#HowtodealwithGCC5.1C11ABIonLinux
-export FGDB_VERSION=1.5.1
-export FGDB_URL=https://github.com/Esri/file-geodatabase-api/raw/master/FileGDB_API_${FGDB_VERSION}/
-
```

Comment:
I think this is now built into the GDAL dependency installed from geoint-deps repo